src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -2022,10 +2022,8 @@ def create_av_set(availability_set_name, resource_group_name,
     if validate:
         return client.validate(resource_group_name, deployment_name, properties)
 
-    LongRunningOperation()(client.create_or_update(
-        resource_group_name, deployment_name, properties, raw=no_wait))
-    compute_client = _compute_client_factory()
-    return compute_client.availability_sets.get(resource_group_name, availability_set_name)
+    return client.create_or_update(
+        resource_group_name, deployment_name, properties, raw=no_wait)
```

Comment:
Yeah, the output does change. It returns the output of the deployment, rather than the output of the availability set. I modified accordingly.